examples/testdata/host_func.go

```diff
@@ -42,6 +42,6 @@ func base64OnString(num uint32) {
 	for i := uint32(0); i < num; i++ {
 		msg := getRandomString()
 		encoded := base64.StdEncoding.EncodeToString([]byte(msg))
-		fmt.Printf("base64 encoded string '%s'\n", encoded[:10])
```

Comment:
anything interesting here? except maybe the massive reduction in wasm :D

Reply:
oh nothing special but funny that after rebuild amd64 has started failing on this due to "out of range" so I just deleted :D

Reply:
it seems like a TinyGo's bug as I see this succeeds after changing this to 
```
		v := encoded[:10]
		fmt.Printf("base64 encoded string '%s'\n", v)
```

Reply:
something buggy on inline slicing :D